bokehjs/src/coffee/core/properties.coffee

```diff
@@ -154,6 +154,10 @@ class Instance extends simple_prop("Instance", (x) -> x.properties?)
 # TODO (bev) separate booleans?
 class Number extends simple_prop("Number", (x) -> _.isNumber(x) or _.isBoolean(x))
 
+# TODO extend Number instead of copying it's predicate
+#class Percent extends Number("Percent", (x) -> 0 <= x <= 1.0)
+class Percent extends simple_prop("Number", (x) -> (_.isNumber(x) or _.isBoolean(x)) and (0 <= x <= 1.0) )
```

Comment:
Note the addition of a `Percent` property here.  Any suggestions  with the TODO would be appreciated.  Or, if people think that the predicate is fine the way it is, I can delete the TODO comment.